app/views/samples_reports/show.haml

```diff
@@ -258,4 +274,30 @@
       document.getElementById("samples-report").classList.remove("hidden");
     });
   }
-  
+
+  // For NIH download button
+
+  var nihDownloadButton = document.querySelectorAll("#print-as-csv");
```

Comment:
Oh, since the element is unique, you can `button = document.getElementById("print-as-csv")` then directly `button.addEventListener(...)`.